quickevent/app/plugins/qml/Runs/src/runswidget.h

```diff
@@ -61,13 +62,26 @@ private slots:
 	QList< QList<int> > runnersByClubSortedByCount(int stage_id, int class_id, QMap<int, QString> &runner_id_to_club);
 	QList<int> runsForClass(int stage_id, int class_id);
 	QMap<int, int> competitorsForClass(int stage_id, int class_id);
+    QList<int> runsForClassCondition(int stage_id, int class_id, QString condition);
```

Comment:
not needed, see QList runsForClass(int stage_id, int class_id, const QString &extra_condition);